### PR TITLE
Feature: M2-2568

### PR DESCRIPTION
**Pull request by moiskillnadne**
Message item type implemented

### CODE REVIEW

File: src/entities/activity/ui/ActivityCardItem.tsx [hunk 1]
```diff
@@ -47,8 +47,10 @@ export const ActivityCardItem = ({
   saveSetAnswerUserEvent,
   saveUserEventByType,
 }: ActivityCardItemProps) => {
+  const isMessageItem = activityItem.responseType === "message"
+
   const buttonConfig: ItemCardButtonsConfig = {
-    isNextDisable: !values || !values.length,
+    isNextDisable: isMessageItem ? false : !values || !values.length,
```
Comment (BamMironov) [inline src/entities/activity/ui/ActivityCardItem.tsx:53]: Please rename the property: isNextDisable => isNextDisabled

File: src/widgets/ActivityGroups/ui/ActivityGroupList.tsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ export const ActivityGroupList = (props: ActivityListWidgetProps) => {
       eventId: activity.eventId,
       progressPayload: {
         ...activityPipelineDetails,
-        startAt: new Date(),
+        startAt: new Date().toString(),
```
Comment (BamMironov) [inline src/widgets/ActivityGroups/ui/ActivityGroupList.tsx:118]: Here also

File: src/entities/activity/ui/ActivityCardItem.tsx (hunk 1 above)
Comment (moiskillnadne) [reply src/entities/activity/ui/ActivityCardItem.tsx:53]: Done

File: src/widgets/ActivityDetails/model/validateItemsBeforeSubmit.ts [hunk 3]
```diff
@@ -14,7 +14,9 @@ export const validateAnswerBeforeSubmit = (
     const isRequired = !item.config.skippableItem
     const isAnswerExist = item.answer[0]
 
-    if (!params?.isAllItemsSkippable && isRequired && !isAnswerExist) {
+    const isMessageItem = item.responseType === "message"
+
+    if (!isMessageItem && !params?.isAllItemsSkippable && isRequired && !isAnswerExist) {
```
[the remark this answers is not in the document]
Comment (moiskillnadne) [reply src/widgets/ActivityDetails/model/validateItemsBeforeSubmit.ts:19]: Done

File: src/widgets/ActivityDetails/ui/ActivityItemList.tsx [hunk 4]
```diff
@@ -66,7 +66,7 @@ export const ActivityItemList = (props: ActivityItemListProps) => {
       eventId,
       activityId: activityDetails.id,
       progressPayload: {
-        endAt: new Date(),
+        endAt: new Date().toString(),
```
[the remark this answers is not in the document]
Comment (moiskillnadne) [reply src/widgets/ActivityDetails/ui/ActivityItemList.tsx:69]: Done

File: src/widgets/ActivityGroups/ui/ActivityGroupList.tsx (hunk 2 above)
Comment (moiskillnadne) [reply src/widgets/ActivityGroups/ui/ActivityGroupList.tsx:118]: Done

**Review by BamMironov — APPROVED** [review]

**Pull request merged**